app/controllers/trainees/contact_details_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Trainees
+  class ContactDetailsController < ApplicationController
+    def index
```

Comment:
I think these actions should be `edit`. I was also wondering whether we should just have one `edit` action on `TraineesController` and just render the correct form for the path? I expect we'll need to treat some of these things separately in any case so no need to worry too much about it now. Just thinkin'.

Reply:
I'm not really fussed. it's kind of transparent. Maybe you kind change them all in one fell swoop in another PR :)